.github/workflows/codecov.yml

```diff
@@ -0,0 +1,26 @@
+name: Coverage
+
+on: [pull_request, push]
```

Comment:
you only need pull_request, because we don't allow changes to the main branch without pull request. So right now the action always runs twice

Reply:
To clarify, "push" doesn't rerun the code coverage if another commit is added to the same existing code request?

Reply:
both are going to run whenever you push to a pull request